docker-compose-debug.yml

```diff
@@ -14,8 +14,8 @@ services:
         akhq:
           security:
             basic-auth:
-              - username: admin
-                password: d49745a74edfd20705ccaa277209db554b6b0d78c20780a4facf87e88aee78c0
+              - username: ${AKHQ_USER}
```

Comment:
Have you tested this locally?  The last time I tried passwords were not picked properly.

Reply:
Try running the UI for this service, and check whether it is asking for authentication

Reply:
It's not asking for authentication, but even putting direct login creds, it's still failing to do so. I guess the problem is somwhere with the container configuration.

Reply:
Haan, figure it out then.